osbs/cli/main.py

```diff
@@ -339,28 +364,22 @@ def cmd_build(args, osbs):
         print("Build skipped")
         return
 
-    build_id = build.get_build_name()
-    # we need to wait for kubelet to schedule the build, otherwise it's 500
-    build = osbs.wait_for_build_to_get_scheduled(build_id)
-    if not args.no_logs:
-        build_logs = osbs.get_build_logs(build_id, follow=True, decode=True)
-        if not isinstance(build_logs, collections.Iterable):
-            logger.error("'%s' is not iterable; can't display logs", build_logs)
-            return
-        print("Build submitted (%s), watching logs (feel free to interrupt)" % build_id)
-        try:
-            for line in build_logs:
-                print('{!r}'.format(line))
-        except Exception as ex:
-            logger.error("Error during fetching logs for build %s: %s", build_id, repr(ex))
+    _print_build_logs(args, osbs, build)
 
-        osbs.wait_for_build_to_finish(build_id)
-        _display_build_summary(osbs.get_build(build_id))
-    else:
-        if args.output == 'json':
-            print_json_nicely(build.json)
-        elif args.output == 'text':
-            print(build_id)
+
+def cmd_build_source_container(args, osbs):
+    build_kwargs = {
+        'user': osbs.build_conf.get_user(),
+        'target': osbs.build_conf.get_koji_target(),
+        'scratch': args.scratch,
+        'sources_for_koji_build_nvr': args.sources_for_koji_build_nvr,
```

Comment:
what about 'sources_for_koji_build_id' ? koji-c will get both nvr & id if provided, so we should get both